corehq/ex-submodules/casexml/apps/phone/xml.py

```diff
@@ -121,11 +121,14 @@ def get_registration_element(restore_user):
     return root
 
 
-def get_registration_element_for_case(restore_user, case_id):
+# Case registration blocks do not have a password
+def get_registration_element_for_case(restore_user, case_id, username):
+    if not username:
+        username = restore_user.username
```

Comment:
can we default this to empty string instead? otherwise the meaning of this field can be confusing